@planetarium/account/src/Account.ts

```diff
@@ -8,6 +8,13 @@ export interface Account {
   sign(message: Message): Promise<Signature>;
 }
 
+export function isAccount(account: unknown): account is Account {
+  return (
+    (account as { publicKey: unknown }).publicKey instanceof PublicKey &&
+    (account as { sign: unknown }).sign instanceof Function
+  );
+}
```

Comment:
I guess this part should be recored in `CHANGES.md` too.

Reply:
I thought it should be fine as-is as `@planetarium/account` is yet to be released. What do you think?

Reply:
I forgot the version on npm is the `dev` release version yet. You're right 👍🏻